### DISCUSSION TITLE
Falling from end void transports to overworld

### DISCUSSION THREAD

**Discussion by 1ByteBit** [PurpurMC/Purpur#1188] [Feature Requests]
(An option, disabled by default ofc) If someone hits the void in the end/right before they start taking void damage, they get transported into the overworld equivalent x/z coords at a possibly configurable height

**Comment by RealRONiN** [comment]
Best suitable as a plugin, which already exists

**Reply by 1ByteBit** [reply]
Yeah I was a bit on the fence about this suggestion myself. It's more of a suggestion for purpur extras then anything
